packages/record-data/addon/-private/graph/-edge-definition.ts

```diff
@@ -77,7 +77,7 @@ function upgradeMeta(meta: RelationshipSchema): UpgradedMeta {
   niceMeta.isCollection = meta.kind === 'hasMany';
   niceMeta.isPolymorphic = options && !!options.polymorphic;
 
-  niceMeta.inverseKey = options && options.inverse;
+  niceMeta.inverseKey = (options && options.inverse) || STR_LATER;
```

Comment:
Whats `STR_LATER`?

Reply:
its defined above, it's a way for us to initialize to an empty string while making it clear we are going to make it non-empty later within the file.